src/script/conversation/MessageRepository.ts

```diff
@@ -1298,18 +1298,18 @@ export class MessageRepository {
       const shouldShowLegalHoldWarning =
         haveNewClientsChangeLegalHoldStatus && updatedLocalLegalHoldStatus === LegalHoldStatus.ENABLED;
 
-      return this.grantMessage(eventInfoEntity, consentType, userIds, shouldShowLegalHoldWarning);
+      await this.grantMessage(eventInfoEntity, consentType, userIds, shouldShowLegalHoldWarning);
     }
 
-    return this.grantMessage(eventInfoEntity, consentType, userIds, false);
+    await this.grantMessage(eventInfoEntity, consentType, userIds, false);
```

Comment:
Ah that's called twice for team users now.

Reply:
Ok, let me put a return after that statement.